include/crocoddyl/multibody/costs/cost-sum.hpp

```diff
@@ -143,12 +143,12 @@ struct CostDataSum {
   Eigen::MatrixXd get_Lxu() const { return Lxu; }
   Eigen::MatrixXd get_Luu() const { return Luu; }
 
-  void set_r(Eigen::VectorXd _r) { r = _r; }
-  void set_Lx(Eigen::VectorXd _Lx) { Lx = _Lx; }
-  void set_Lu(Eigen::VectorXd _Lu) { Lu = _Lu; }
-  void set_Lxx(Eigen::MatrixXd _Lxx) { Lxx = _Lxx; }
-  void set_Lxu(Eigen::MatrixXd _Lxu) { Lxu = _Lxu; }
-  void set_Luu(Eigen::MatrixXd _Luu) { Luu = _Luu; }
+  void set_r(const Eigen::VectorXd& _r) { r = _r; }
```

Comment:
I believe we should add size checks here for all following methods

Reply:
Done. Thanks for the feedback